msg/timesync.msg

```diff
@@ -1,4 +1,8 @@
 uint64 timestamp		# time since system start (microseconds)
+uint8 seq                       # timesync msg sequence
```

Comment:
Can you just make this a new message instead of hijacking the metadata/status message?

Reply:
Yes. This is wip and I am finding a better solution to keep all the implementation consistent.